src/django/api/migrations/0141_delete_contributor_webhooks.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 3.2.4 on 2024-03-06 15:49
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0140_remove_indexing_ppe_fields'),
+    ]
+
+    operations = [
+        migrations.DeleteModel(
```

Comment:
@Innavin369 As we discussed on 1 by 1 call, did you plan to remove creation of this table and just create sql script that will delete the table if exist for this migration?

Reply:
I've also discussed this with Vlad and we decided not to touch previous migrations left as it is and add migration for removing the table.